### ISSUE TITLE
Tab Switcher Skips Last Tab

### ISSUE THREAD

**Issue by Trcx528** [oggy/tab-switcher#18]
I just upgraded to 1.2.0 this morning and it feels a little "derpy."  I think I've narrowed it down to the switcher skipping the last used tab.  

![image](https://cloud.githubusercontent.com/assets/1213427/8441927/230cc000-1f41-11e5-9d08-45c75db117f8.png)

In the above image I was editing PING.php, and switched to NETDOWN.php.  Hitting the hotkey and bringing up the switcher skips immediately to index.php.  I can press the hotkey again to cycle all the way around, and it will once again skip over PING.php.  In short, the last used tab is not selectable. I am unsure of what could cause this.  This bug is not present in previous version.

**Comment by oggy** [comment]
Hi Trcx,

So you're saying when you hit `alt-]`, it starts on the 3rd item? (It's supposed be the 2nd, with the 1st being the current tab.) I'm unable to reproduce this, either on Mac OS, or a WIn 8.1 VM. Both using Atom 1.0. You could try v1.2.1, but I don't *think* anything there would change this.

Does it still happen after reloading Atom (`window:reload`)? If so, then what if you move aside your config directory (i.e. start clean), and only install tab-switcher?

**Comment by Trcx528** [comment]
That is correct. I remapped the hotkeys to the standard tab switch keys (ctrl tab, ctrl shift tab), but I doubt that'd cause any issues.  

Regardless it seems to have been resolved this morning when I updated to 1.2.1, not sure what changed but I'm just happy to have it working as expected.  Thanks for your help.

In the event that this is reopened in the future here are a list of installed packages:
* Remote-FTP @ 0.7.1
* Aligner-php @ 1.0.0
* Atom-autocomplete-php @ 0.1.10
* Color-picker @ 2.0.7
* File-icons @ 1.5.8
* HIghlight-line @ 0.11.0
* Minimap @ 4.10.1
* Quick-query @ 0.4.0
